client/lib/analytics/signup.js

```diff
@@ -123,16 +119,14 @@ export function recordSignupComplete(
 	gaRecordEvent( 'Signup', 'calypso_signup_complete:' + flags.join( ',' ) );
 
 	// Tracks, Google Analytics
-	if ( isNew7DUserSite ) {
+	if ( isNewSite && isNewUser ) {
 		const device = resolveDeviceTypeByViewPort();
 
 		// Tracks
 		recordTracksEvent( 'calypso_new_user_site_creation', {
 			flow,
 			device,
 			framework,
-			is_new_user: isNewUser,
-			is_newish_user: isNewishUser,
```

Comment:
Just double checking that this is intentional.

Reply:
Well, if we go with this new version, `is_new_user` will always be true, and we are not using `is_newish_user` anymore.

Let's hear what @MicBosi thinks!

Reply:
Yes indeed! I also expected to remove these two fields which were added [recently](https://github.com/Automattic/wp-calypso/commit/4fca5b760077f24cf50584cbc9e7ff6bec2cf4b5#diff-a394231df3a1854c3401bf55c39967a6c23e535d36f0436bf99d795f2bd7b29dR128) and I don't think are actually being used by us in data, perhaps they were more useful for debugging - and now we don't need them because the `if` condition.